tests/index.js

```diff
@@ -497,11 +497,61 @@ describe('ember-cli-version-checker', function() {
               },
 
               function(err) {
-                return err.suppressStacktrace;
+                return !err.stack;
               }
             );
           });
         });
+
+        describe('Multiple instances check', function() {
+          beforeEach(function() {
+            projectRoot.write({
```

Comment:
can we use: https://github.com/stefanpenner/node-fixturify-project as it may be easier to maintain.